charts/icm-web/.bumpversion.cfg

```diff
@@ -0,0 +1,13 @@
+[bumpversion]
+current_version = 0.1.30
+commit = true
+tag = false
+message = Bump icm-web chart version: {current_version} → {new_version}
+
+[bumpversion:file:Chart.yaml]
+search = version: {current_version}
+replace = version: {new_version}
+
+[bumpversion:file:README.md]
```

Comment:
Same here. The `README.md` section is obsolete.

Reply:
done